delete platform code and workflows

c-p-b opened the pull request:
## What
This code now lives in [airbytehq/airbyte-platform](https://github.com/airbytehq/airbyte-platform)

Commit:
delete platform code and workflows

Commit:
delete more platform workflows

Commit:
sunset kustomize (#23187)

* sunset kustomize

* remove build steps that use kustomize

* remove k8s kustomize gke-kube-acceptance-tests

Commit:
delete docker-compose files

Commit:
Merge branch 'master' into platform-split

cpdeethree commented:
Octavia CLI build needs the shell script from here added in order to run docker-compose: https://github.com/airbytehq/monorepo/issues/32

Commit:
remove platform from check_images_exist.sh

supertopher committed:
Working replacement script for docker

Commit:
rename start script and add download files to gitignore

Commit:
add args and download only option

Commit:
integration test downloads docker compose files

Commit:
remove helm charts

Commit:
remove more platform files

Commit:
add flags.yml

Commit:
use version from gradle.properties instead of .env

supertopher committed:
Working replacement script for docker (#23227)

* Working replacement script for docker

* Working debugging line

* rename start script and add download files to gitignore

* add args and download only option

* Made flags cleaner and more cleanup

i've never used this flag pattern, its much
easier than my usual way

* warn about a sharp edge

* Don't run when we don't know a command

* better help spacing

* add flags.yml

---------

Co-authored-by: cpdeethree <conor@airbyte.io>
Co-authored-by: Conor <cpdeethree@users.noreply.github.com>

Commit:
remove more-secrets env

froblesmartin commented:
It would be nice to leave a note in the README.md so that people can easily spot where the files are now (I was searching for the Helm charts, as in artifacthub there is no "source code" link :(

cgardens commented:
thanks for the note. @cpdeethree has updated the description to help with that!
